### PR TITLE
CORS

### LINKED ISSUE #39: Update API CORS

### PR DESCRIPTION
**Pull request by sagemlee**
### Description
Added in CORS headers for our queries/mutations
Added a test for CORS

### Type of change
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Refactor (non-breaking change which maintains existing functionality)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: spec/requests/v1/cors_spec.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+describe "the requests support CORS headers" do
+  it 'Returns the response CORS headers' do
+
+    hello = Phrase.create(expression: "Hello",
+                    image: "blahblahblah.url",
+                    )
+
+    post '/v1/graphql', params: { query: query(id: hello.id)}, headers: {'ORIGIN' => '*'}
```
Comment (sagemlee) [inline spec/requests/v1/cors_spec.rb:10]: This is basically the same test as in our phrase query spec, just needed something to make sure the correct header was coming through.

Comment (cgaddis36) [reply spec/requests/v1/cors_spec.rb:10]: Great testing here though, targets exactly what we need it to

Comment (EricLarson2020) [reply spec/requests/v1/cors_spec.rb:10]: Nice work making a test for this

File: Gemfile [hunk 2]
```diff
@@ -20,6 +20,7 @@ gem 'graphql'
 gem 'sass-rails'
 gem 'uglifier'
 gem 'coffee-rails'
+gem 'rack-cors'
```
Comment (cgaddis36) [inline Gemfile:23]: nifty lil gem

File: config/application.rb [hunk 3]
```diff
@@ -31,5 +31,16 @@ class Application < Rails::Application
     # Middleware like session, flash, cookies can be added back manually.
     # Skip views, helpers and assets when generating a new resource.
     config.api_only = true
+
```
Comment (EricLarson2020) [inline config/application.rb:34]: good work finding this

**Review by EricLarson2020 — APPROVED** [review]

**Review by cgaddis36 — APPROVED** [review]
Great work setting up CORS for the FE team

**Pull request merged**